cmd/suggestion/hyperopt/v1beta1/Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV TARGET_DIR /opt/katib
 ENV SUGGESTION_DIR cmd/suggestion/hyperopt/v1beta1
 ENV PYTHONPATH ${TARGET_DIR}:${TARGET_DIR}/pkg/apis/manager/v1beta1/python:${TARGET_DIR}/pkg/apis/manager/health/python
 
-RUN if [ "${TARGETARCH}" = "ppc64le" ] || [ "${TARGETARCH}" = "aarch64" ]; then \
+RUN if [ "${TARGETARCH}" = "ppc64le" ]; then \
```

Comment:
BuildKit propagates `TARGETARCH` with `amd64` or `arm64`. However, we have been able to build container images for the platform linux/arm64 with an incorrect value, `aarch64`.

This means we don't need these packages for the platform `linux/arm64`.